src/server/scripts/Kalimdor/CavernsOfTime/BattleForMountHyjal/hyjal_trash.cpp

```diff
@@ -825,6 +830,12 @@ class npc_necromancer : public CreatureScript
 
         void JustEngagedWith(Unit* /*who*/) override { }
 
+        void JustDied(Unit* killer) override
+        {
+            hyjal_trashAI::JustDied(killer);
+            me->AI()->Talk(TRASH_SAY_DEATH);
```

Comment:
Just Talk(TRASH_SAY_DEATH); ?

Reply:
Any function to handle it in db?I can't find a way.

Reply:
Yes, in SAI. But I doubt it's a good idea to have both SAI and C++ at once

Reply:
Both me->AI()->Talk can be just changed to Talk(BLAH_BLAH);